lib/wit/main.py

```diff
@@ -108,6 +111,37 @@ def main() -> None:
         raise WitBug(e)
 
 
+def foreach(ws, args):
+    packages = sorted(ws.lock.packages, key=lambda x: str.lower(x.name))
+
+    has_fail = False
+    for pkg in packages:
+        env = os.environ.copy()
+        env["WIT_REPO_NAME"] = pkg.name
+        env["WIT_REPO_PATH"] = str(ws.root / pkg.name)
```

Comment:
Is this the same as `pkg.repo.path`?

Reply:
Just tried it, it's value is `None`, I'm guessing we never create the repo object

Reply:
Yeah this is one of the weird parts of the implementation, I *think* `pkg.repo` is set by running `resolve` or something like that. We should probably set it for the `Package`s in the Lockfile though when we construct the `Workspace` object.